docs/New Game.md

```diff
@@ -152,3 +152,28 @@ misconfiguration in Randovania.
 Before a game can work in Multiworld, it must first work properly as a solo game. It is highly recommended to finish the basic integration first.
 
 **TODO**
+
+# Expectations of a Development Game
+
+After a game has been merged and is available in development builds, the responsibility of maintaining it is now shared between you and the Randovania team:
+- Randovania: We make sure your game is included in all Randovania changes.
+- You: Work towards the game reaching stable.
+
+We're happy to support new games by doing our part, but since that has a cost, that we don't want to be doing forever, there are some expectations regarding time frames:
+
+### Exportable from Dev Builds
+
+After roughly one year, we expect a game to be exportable directly from a regular dev build.
+How good the experience is when playing a seed is up to your discretion.
+We do however recommend integrating it as early as it is possible, for a better developer experience.
+
+### Becoming Stable
+
+After roughly two years, we expect a game to be close to being considered stable.
+
+### Active Development
+
+Please don't go for too long without visible development of either patcher or non-logic Randovania changes.
```

Comment:
this should be a bit more specific and mention that this is primarily for *before* going stable. once a game is stable, by nature further active development isn't really needed.

Reply:
I'd also add a line explicitly mentioning that games which aren't in active development may be subject to removal if they don't meet the time frames mentioned above

Reply:
Good point on stable. Any suggestions on the later?